src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import { useState } from 'react';
+import { ErrorTypes } from '../../types/ErrorTypes';
+
+type Props = {
+  error: ErrorTypes | null,
+};
+
+export const Notification: React.FC<Props> = ({ error }) => {
+  const [onButtonDelete, setOnButtonDelete] = useState(true);
```

Comment:
Consider renaming 'onButtonDelete' to 'isNotificationVisible' to better reflect its purpose.

Reply:
updated